### DISCUSSION TITLE
Sanity/cli error while deploying Next.js app + Sanity Studio

### DISCUSSION THREAD

**Discussion by arsekil** [vercel/community#1252] [Help]
### Summary

Hello all,

I always get an error of `missing module of @sanity/cli or it type declarations`, while I want to deploy my next.js app including the sanity studio.
![sanityCLIerror](https://user-images.githubusercontent.com/20362038/212030363-dcc983b0-2047-4010-a96f-509012d8b1f0.png)

I installed sanity-cli globally, but it didn't help. Is there a way to solve this?

Thank you in advance if you reply regarding the topic.

**Comment by amyegan** [comment]
Sorry you're running into an error @andrejmoltok. Can you tell me more about your deployment workflow? I want to be sure I'm looking in the right place.

**Reply by arsekil** [reply]
Well, I'm not sure what that means, but I have a combination of Next.js and Sanity Studio installed on my laptop and then sent to Github. From there I imported it into Vercel and without any other modifications I hit Deployment button and got this error. It seems that the compilation of Next.js is all right, but only when it gets to Sanity Studio, it halts right there.

**Comment by HEVpro** [comment]
I have the same issue as @andrejmoltok. I move my project to v2 to v3 and when redeploy my project it doesn't work. :(

**Comment by amyegan** [comment]
I can't say exactly what is going wrong without digging into a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) or logs. But there is an up-to-date template provided by Sanity that uses Sanity Studio that could be used as a reference for debugging - [Personal Website with Built-in Content Editing](https://www.sanity.io/templates/personal-website-with-built-in-content-editing)
